java/core/src/main/java/co/worklytics/psoxy/rules/Rules2.java

```diff
@@ -21,7 +21,7 @@
 @EqualsAndHashCode
 @JsonPropertyOrder({"allowAllEndpoints", "endpoints", "defaultScopeIdForSource"})
 @JsonInclude(JsonInclude.Include.NON_NULL) //NOTE: despite name, also affects YAML encoding
-public class Rules2 implements RuleSet, Serializable {
+public class Rules2 implements RESTRules {
```

Comment:
this is clearer; lets people bind to `RESTRules` interface instead of `Rules2` implementation

Reply:
Rules2 --> PsoxyRules